PrebidMobile/Swift/AdUnits/AdUnit.swift

```diff
@@ -319,6 +319,18 @@ public class AdUnit: NSObject, DispatcherDelegate {
         adUnitConfig.impORTBConfig
     }
     
+    /// Sets the content OpenRTB configuration string for the ad unit.
+    ///
+    /// - Parameter ortbConfig: The content OpenRTB configuration string to set. Can be `nil` to clear the configuration.
+    public func setContentORTBConfig(_ ortbConfig: String?) {
```

Comment:
I'd suggest introducing the full-featured `setGlobalOrtbConfig` on the ad unit level. The `Targeting.setGlobalOrtbConfig` method should continue to work without changes. But if the publisher sets the unit level `setGlobalOrtbConfig`, it should take precedence over `Targeting.setGlobalOrtbConfig` .

We can also consider "merging" of AdUnit and the Targeting level global config. 

Thoughts?